src/client_side_reply.cc

```diff
@@ -1528,6 +1528,9 @@ clientReplyContext::identifyStoreObject()
 
     // client sent CC:no-cache or some other condition has been
     // encountered which prevents delivering a public/cached object.
+    // XXX: The above text does not match the condition below. It might describe
```

Comment:
IMO this can and should be fixed by this PR. Reversing the if-else like below should be correct.
```
@@ -1528,6 +1528,9 @@ clientReplyContext::identifyStoreObject()

    if (r->flags.noCache) {
        // client sent CC:no-cache which prevents delivering a public/cached object.
        identifyFoundObject(nullptr, "no-cache");
    } else {
        const auto e = storeGetPublicByRequest(r);
        identifyFoundObject(e, storeLookupString(bool(e)));
    }
```

Reply:
> the titular scope/purpose of this PR is to make Squid obey CC:no-cache on manager requests.

I see no such scope/purpose or even a hint at such scope/purpose! This PR is about making new cache manager responses uncachable and preventing previously cached responses from being served from the cache. That is all. This PR changes do not (and should not) care about Cache-Control headers in cache manager requests! What can I change in PR description to clarify that?

Reply:
@yadij, please move this bug-fixing PR forward. The current basic disagreement related to this added `XXX` comment should not take weeks to resolve, but it has been more than two weeks since your last response. Again, if it helps, I will remove that `XXX` comment.

Reply:
@yadij, please move this bug-fixing `review-1` PR forward. It has been waiting for your review for a month already.

Reply:
@yadij, please move this bug-fixing `review-1` PR forward. It has been waiting for your review for two months already.

Reply:
Issue worked around by [c380732](https://github.com/squid-cache/squid/pull/1185/commits/c38073237e29e0bafb3520c418da57625ad9c5ac).